apple/RNCWebView.m

```diff
@@ -64,6 +64,29 @@ - (void)scrollWheel:(NSEvent *)theEvent {
 @end
 #endif // TARGET_OS_OSX
 
+#if !TARGET_OS_OSX
+@interface RNCWKWebView : WKWebView
+@end
+@implementation RNCWKWebView
+- (BOOL) canPerformAction:(SEL)action withSender:(id)sender
+{
+  NSString *sel = NSStringFromSelector(action);
+  RNCWebView *rncWebView = (RNCWebView *)[self superview];
+  RCTAssert([rncWebView isKindOfClass:[rncWebView class]], @"superview must be an RNCWebView");
+  
+  if(![rncWebView allowTextStyleOptions]) {
+    NSArray* textStyleOptions = @[@"toggleBoldface:", @"toggleItalics:", @"toggleUnderline:"];
```

Comment:
Perhaps this could be made more flexible by taking the array of options to remove as a property? So instead of `allowTextStyleOptions` it would be something like `suppressMenuItems={['toggleBoldFace', 'toggleItalics']}` (or `removeMenuItems`... or `menuItemsToRemove`... whatever) ?

Reply:
Yeah, sure! Looks like a good option. I will just close this PR and create a new one, since this is really outdated now en the feature changes a lot.